spritz/app/src/main/java/com/novoda/spritz/sample/MainActivity.java

```diff
@@ -0,0 +1,122 @@
+
+        LottieAnimationView lottieAnimationView = findViewById(R.id.animation_view);
+        lottieAnimationView.setImageAssetsFolder("images");
+
+        final ViewPager viewPager = findViewById(R.id.viewpager);
+        viewPager.setAdapter(new ScreenSlidePagerAdapter(getSupportFragmentManager()));
+
+        spritz = Spritz
+                .with(lottieAnimationView)
+                .withDefaultSwipeAnimationDuration(300, TimeUnit.MILLISECONDS)
+                .withDefaultSwipeForwardInterpolator(SWIPE_FORWARD_INTERPOLATOR)
+                .withDefaultSwipeBackwardsInterpolator(SWIPE_BACKWARDS_INTERPOLATOR)
+                .withPages(
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                                .build(),
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                                .build(),
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .build()
+                )
+                .attachTo(viewPager);
+
+        Button continueButton = findViewById(R.id.btn_continue);
+        continueButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int nextItem = viewPager.getCurrentItem() + 1;
+                if (PAGER_NUM > nextItem) {
+                    viewPager.setCurrentItem(nextItem);
+                }
+            }
+        });
+        Button previousButton = findViewById(R.id.btn_previous);
+        previousButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int prevItem = viewPager.getCurrentItem() - 1;
+                if (prevItem >= 0) {
+                    viewPager.setCurrentItem(prevItem);
+                }
+            }
+        });
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        spritz.startPendingAnimations();
+    }
+
+    private class ScreenSlidePagerAdapter extends FragmentStatePagerAdapter {
+
+        ScreenSlidePagerAdapter(FragmentManager fragmentManager) {
+            super(fragmentManager);
+        }
+
+        @Override
+        public Fragment getItem(int position) {
+            return new AnimationFragment();
+        }
+
+        @Override
+        public int getCount() {
+            return PAGER_NUM;
+        }
+
+    }
+
+    public static class AnimationFragment extends Fragment {
+
+        public AnimationFragment() {
+            // default constructor
+        }
+
+        @Nullable
```

Comment:
This is not true

Reply:
Done in efe508669bc9b38523311c9def54b54435736883.